src/components/Camera/measure_blur.js

```diff
@@ -0,0 +1,98 @@
+//ref: https://github.com/timotgl/inspector-bokeh/blob/master/measure_blur.js
```

Comment:
I'm not sure this is the best source for doing something like this

Personally I would have used something more similar to these methods http://stackoverflow.com/questions/7765810/is-there-a-way-to-detect-if-an-image-is-blurry so the third-party code that you bring in is trying to do something pretty simple and well-defined (e.g. get the laplacian transform, or fast fourier transform)

And the logic that you're bringing into the direct source code is quite minimalist in comparison.

But I don't know - is there a reason you chose this specific "measure blur" source?